docs/api/client/http-api.md

```diff
@@ -6,6 +6,65 @@ description: ReSolve provides a standard HTTP API that allows you to send aggreg
 
```

Comment:
Line 4 (above):
`description: ReSolve provides a standard HTTP API that allows you to send aggregate commands, and query Read and View Models.`

Please rewrite without using "provides".

Reply:
Changed to "ReSolve exposes an HTTP API that allows you to send aggregate commands, and query Read and View Models."